common/src/Model/Brush.cpp

```diff
@@ -1273,7 +1273,7 @@ namespace TrenchBroom {
         }
 
         void Brush::rebuildGeometry(const vm::bbox3& worldBounds) {
-            const vm::bbox3 oldBounds = bounds();
+            const vm::bbox3 oldBounds = cullingBounds();
             deleteGeometry();
             buildGeometry(worldBounds);
             nodeBoundsDidChange(oldBounds);
```

Comment:
Can we not ditch the old bounds here, too, now that we don't need them to update the AABB tree? Because if we don't, we'll have to rename these methods to `nodeCullingBoundsDidChange`, too.

Reply:
Looks like `Node::childBoundsDidChange` needs the old bounds for an optimization, where it avoids calling this->nodeBoundsDidChange unless the child contributed before, and doesn't now.

So it looks non trivial to remove the oldBounds param. I think I'll just rename things as needed rather than try to simplify the bounds changed notifications in this PR, sound ok?

Reply:
Yes!